### DISCUSSION TITLE
The save button does not appear in the block explorer module

### DISCUSSION THREAD

**Discussion by rogeriojpp** [Budibase/budibase#12393] [Help] [answered]
The save button does not appear in the block explorer module on the Budibase Cloud

**Comment by ConorWebb96** [comment, marked as answer]
Hey @rogeriojpp,

This isn't just an issue with cloud it also seems to be occurring on self-host as well. I think i know what the issues is, whenever I ejected the explore block component, there was styling added to the button itself. I've shown a screenshot of this below.

<img width="1512" alt="Screenshot 2023-11-20 at 08 59 42" src="https://github.com/Budibase/budibase/assets/126772285/8f7ede15-c057-4977-ac1a-9a2115fea759">

Removing the below code after ejecting the block will fix the button display issue;
```handleBars
{{#if (not Device.Mobile) }}
  display: none;
{{/if}}
```

I'll create an issue for this on Github and link it here later.

Thanks for bringing this to our attention, hopefully this helps.

**Comment by rogeriojpp** [comment]
![image](https://github.com/Budibase/budibase/assets/88006639/231d3505-7fea-499e-9013-70d7690281c9)
![image](https://github.com/Budibase/budibase/assets/88006639/e7af8eb0-0f61-496f-b770-ea7c84c15387)

I performed the procedures on both self-hosted Budibase and Budibase cloud and it didn't resolve the problem in any way. Could this be because of the latest Budibase update? If so, how do I downgrade to a version where the button appears? Below is a print of the changes in question.

**Reply by ConorWebb96** [reply]
Hey @rogeriojpp, 

I'm not quite sure what you mean here, what you are showing on the left and box in the middle are the connected areas?

Possibly try deleting the component and re-adding the block again, then ejecting. This may clear up any issues you are having.

Cloud version, were i've removed the styling and its showing the back button.
<img width="1511" alt="Screenshot 2023-11-20 at 11 06 16" src="https://github.com/Budibase/budibase/assets/126772285/3740be89-45cb-437b-8a8a-4829f57fd8af">
<img width="1418" alt="Screenshot 2023-11-20 at 11 06 31" src="https://github.com/Budibase/budibase/assets/126772285/7757f2c6-ff02-4683-b427-a0c2a95b4661">

Self-host version, were i've removed the styling and its showing the back button.
<img width="1512" alt="Screenshot 2023-11-20 at 11 05 01" src="https://github.com/Budibase/budibase/assets/126772285/1bca8a12-7946-40e3-9258-6690d1da5ce7">
<img width="1419" alt="Screenshot 2023-11-20 at 11 05 14" src="https://github.com/Budibase/budibase/assets/126772285/20e3085f-4bf9-495d-8e49-bc521c3889db">

Both of the above screenshots are using the latest versions of Budibase. After removing the styling on the button it seemed to fix itself.

I hope this helps.
